Description: Put all orphan files into the airportlinkmon package
 Some files were not declared as being part of a package.
Author: Julien BLACHE <jblache@debian.org>

Index: airport-utils-2/linkmon/AboutDialog.java
===================================================================
--- airport-utils-2.orig/linkmon/AboutDialog.java	2001-05-14 21:49:04.000000000 +0200
+++ airport-utils-2/linkmon/AboutDialog.java	2010-03-11 20:56:30.794753126 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airportlinkmon;
 
 import java.awt.*;
 import javax.swing.*;
Index: airport-utils-2/linkmon/LinkMonitor.java
===================================================================
--- airport-utils-2.orig/linkmon/LinkMonitor.java	2001-05-14 21:50:00.000000000 +0200
+++ airport-utils-2/linkmon/LinkMonitor.java	2010-03-11 20:56:30.807751991 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airportlinkmon;
 
 import java.util.*;
 import java.net.*;
